docs/source/main_classes/pipelines.rst

```diff
@@ -45,12 +45,50 @@ The pipeline abstraction
 The `pipeline` abstraction is a wrapper around all the other available pipelines. It is instantiated as any other
 pipeline but requires an additional argument which is the `task`.
 
+Simple call on one item: .. code-block::
+
+    >>> pipe = pipeline("text-classification") >>> pipe("This restaurant is awesome") [{'label': 'POSITIVE', 'score':
+    0.9998743534088135}]
```

Comment:
There should be a line return here no?
```suggestion
    >>> pipe = pipeline("text-classification") 
    >>> pipe("This restaurant is awesome") 
    [{'label': 'POSITIVE', 'score':
    0.9998743534088135}]
```

Reply:
Yes, I must have made a mistake and auto doc formatting edited into a worse shape, fixed.